src/main/java/com/google/api/codegen/viewmodel/OptionalArrayMethodView.java

```diff
@@ -58,11 +58,18 @@
 
   public abstract String stubName();
 
-  public abstract boolean isLongRunningOperation();
-
   @Nullable
   public abstract LongRunningOperationDetailView longRunningView();
 
+  public boolean isLongRunningOperation() {
+    return longRunningView() != null;
+  }
+
+  public boolean isNotGrpcRequestStreamingMethod() {
+    return !(grpcStreamingType().equals(GrpcStreamingType.BidiStreaming)
+        || grpcStreamingType().equals(GrpcStreamingType.ClientStreaming));
+  }
```

Comment:
Since the logic is non-trivial here, I think I would prefer to have it in the transformer. Also, maybe the concept should be more general: `isSingularRequestMethod` or something like that.